### ISSUE TITLE
Zeus crashes on Activity page

### ISSUE THREAD

**Issue by iWarpBTC** [ZeusLN/zeus#2201]
### Describe the bug

I've got Zeus linked to a LNbits wallet using Lndhub. There's an incoming payment created by lnurlp extension with comments turned on. Zeus crashes when I'm on the Activity page and try to open detailts of this specific payment. 

### Reproduce

- In a lnbits wallet enable lndhub and lnurlp ext. 
- Create a new pay link and allow comments (advanced options).
- Pay to this link
- Connect Zeus to this wallet using lndhub
- In Activity open details of the payment

### ZEUS version

v0.8.4

### Node interface

LndHub

### Network

Clearnet

### Device

cross-platform

### Device operating system

cross-platform

**Comment by kaloudis** [comment]
Here's the payment object in question. Issue appears to stem from the description returned as an array:

```
{
    "add_index":"500",
    "amt":20,
    "description":[
        "Test"
    ],
    "expire_time":1716304559,
    "formattedTimeUntilExpiry":"Never",
    "ispaid":true,
    "payment_hash":"66f7ffcf9c1d50fef2ce5bcf0452d751edd305b60de16ecd756e9f09cda9907e",
    "payment_request":"lnbc200n1pnye254sp5v2p833deas4emnd0wzqepzkuy8cndr559h2fy9cqajyuxesndm0spp5vmmllnuur4g0aukwt08sg5kh28kaxpdkphskant4d60snndfjplqhp56j56l8d7cqpd83kxnnzd586rjsh5qkzftu5jvm5y52xy7c8m7rfscqpjrzjq073qhta7ev46k3k7qtt3luu5a89ga8zjkfufwagtaxh44rx80z7ur9qd5qq8pqqqyqqqqlgqqqqqqgq9q9qxpqysgqd5sxp08jsaupu33rft9n6vh0sg0w45pf825s8tf8ykmju5vqkt5kgea5el92fjpjrlmjlmd7d8ku0s67n4nmqpyl60msw7zmxr73cegp4pm4c9",
    "r_hash":{
        "data":[
            Array
        ],
        "type":"Buffer"
    },
    "timestamp":1716300437,
    "type":"user_invoice"
}
```

**Comment by kaloudis** [comment]
This has been fixed in v0.8.5
